waku/v2/node/storage/migration/migrations_scripts/message/00003_deleteReceiverTimeStamp.up.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS Message_backup (
+        id BLOB PRIMARY KEY,
+        contentTopic BLOB NOT NULL,
+        pubsubTopic BLOB NOT NULL,
+        payload BLOB,
+        version INTEGER NOT NULL,
+        senderTimestamp REAL NOT NULL
+    ) WITHOUT ROWID;
+
+INSERT INTO Message_backup SELECT id, contentTopic, pubsubTopic, payload, version, senderTimestamp  FROM Message;
+
+DROP TABLE Message;
+
+CREATE TABLE IF NOT EXISTS Message(
+        id BLOB PRIMARY KEY,
+        contentTopic BLOB NOT NULL,
+        pubsubTopic BLOB NOT NULL,
+        payload BLOB,
+        version INTEGER NOT NULL,
+        senderTimestamp REAL NOT NULL
+    ) WITHOUT ROWID;
+
+
+INSERT INTO Message SELECT id, contentTopic, pubsubTopic, payload, version, senderTimestamp  FROM Message_backup;
```

Comment:
Wondering for interest's sake what the benefit is of doing it this way (creating a new table based on the previous) vs just deleting the column (`ALTER TABLE...DROP COLUMN`)? This will still involve making a backup copy, of course.

Reply:
Good question, unfortunately, SQlite does not support the `ALTER TABLE` command